rplugin/python3/iron/repls/php.py

```diff
@@ -0,0 +1,15 @@
+# encoding:utf-8
+"""PHP repl definition for iron.nvim. """
+from iron.repls.utils.cmd import detect_fn
+
+php = {
+    'command': 'php -a',
+    'language': 'php',
+    'detect': detect_fn('php'),
+}
+
+psyshell = {
+    'command': 'psysh',
+    'language': 'php',
+    'detect': detect_fn('php'),
```

Comment:
I'm terribly sorry for the lack of a decent documentation.

There order matters, this is true, but Iron will check for the binary using the provided function on the 'detect' key.

It works for you because you have `psysh` installed, but it'll falsely accuse it a valid repl option for those who don't, since it's checking for the wrong binary.